CHANGELOG.md

```diff
@@ -43,7 +44,7 @@ No changes since 0.91.1.
 
 ### Bug fixes
 
-* Fixed a bug when opening a Realm cause a staled memory mapping. Symptoms are error messages like "Bad or incompatible history type", "File format version doesn't match", and "Encrypted interprocess sharing is currently unsupported". 
+* Fixed a bug when opening a Realm cause a staled memory mapping. Symptoms are error messages like "Bad or incompatible history type", "File format version doesn't match", and "Encrypted interprocess sharing is currently unsupported".
```

Comment:
`Fixed a bug when opening a Realm cause a staled memory mapping.`
`staled` -> `stalled`
`causes` seems more proper, but I am not good at English.